CHANGELOG.md

```diff
@@ -20,6 +22,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - User experience for situations where Conjur identity cannot be resolved has been
   improved with better errors/warnings.
   [cyberark/conjur-puppet#125](https://github.com/cyberark/conjur-puppet/issues/125).
+- Added warnings and better error logic to certificate parsing logic.
+  [cyberark/conjur-puppet#115](https://github.com/cyberark/conjur-puppet/issues/115).
+
+### Fixed
+- Fixed account name escaping during variable fetching.
```

Comment:
is there any more detail on this? what might have happened before?

looking for wording like

"Account names with special characters [example] are now properly escaped when reading the configuration for this module" or something